lib/server.js

```diff
@@ -38,6 +34,18 @@ var TEMPLATE = ['<script id="%s" type="application/javascript">var ',
 
 exports.create = function create(createOptions) {
 
+  // safely require the peer-dependencies
+  var React = util.safeRequire('react');
+  var Router = util.safeRequire('react-router');
+
+  if (!React) {
+    throw new Error('React is required as a peer dependency. Run `npm instal react`.')
+  }
+
+  if (!Router) {
+    throw new Error('React Router is required as a peer dependency. Run `npm instal react-router`.')
```

Comment:
It is not? https://github.com/paypal/react-engine/issues/97

> @stevemao right now no. I guess we can move this https://github.com/paypal/react-engine/blob/master/lib/server.js#L29

Sorry if I misunderstood you.